spec/concepts/personne_morale/operation/update_spec.rb

```diff
@@ -19,22 +19,50 @@
   end
 
   context 'when the dossier is found' do
+    context 'when the personne morale exists' do
+      before do
+        create(
+          :dossier_entreprise_with_pm,
+          code_greffe: data[:code_greffe],
+          numero_gestion: data[:numero_gestion],
+        )
+      end
+
+      it { is_expected.to be_success }
+
+      it 'updates the given fields' do
```

Comment:
ici se cachent deux tests, la persistance et la cohérence des valeurs. Ok pour cette solution ?

Reply:
J'ai trouvé que c'était un bon trade-off, j'aurais pu rajouter un `expect(updated_pm).to be_persisted` oui, mais visiblement tu en as pas eu besoin pour voir le test de persistance ^^